packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -960,7 +965,7 @@ export class ContextualMenuBase extends React.Component<IContextualMenuProps, IC
     let handled = false;
 
     if (shouldHandleKey(ev)) {
-      this._focusingPreviousElement = false;
+      this._focusingPreviousElement = true;
```

Comment:
this change looks wrong to me?

Reply:
This was broken before. The original was setting it to true, I set it to false here incorrectly in the previous change.